tox.ini

```diff
@@ -116,7 +116,7 @@ commands_pre =
     pip uninstall -y pyautogui pytest-qt pyqt5 pyside2 pyside6 pyqt6
 
 commands =
-    cov: coverage run \
```

Comment:
@jni I have moved to triggering parallel mode here. It simplifies local coverage calculation. I may move this to a separate PR.

Reply:
Yeah I would prefer that. It makes it easier to think about this PR — I was wondering if there was some change in the parallelism constraints given that there is some new threading code.